presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java

```diff
@@ -86,11 +86,14 @@ private static HiveTableDefinition allHiveSimpleTypesParquetTableDefinition()
                         "   c_bigint             BIGINT," +
                         "   c_float              FLOAT," +
                         "   c_double             DOUBLE," +
+                        "   c_decimal            DECIMAL," +
+                        "   c_decimal_w_params   DECIMAL(10,5)," +
                         "   c_timestamp          TIMESTAMP," +
                         "   c_string             STRING," +
                         "   c_varchar            VARCHAR(10)," +
                         "   c_char               CHAR(10)," +
```

Comment:
@cawallin: I will leave the `INSERT` test quarantined for now as you are playing with that as part of `FLOAT`. Probably we should (at least for now) split table definitions for `SELECT` and `INSERT` tests as both flows support different subset of types.